requestor-tutorials/service-development/service-example-2-managed-erigon.md

```diff
@@ -38,6 +38,127 @@ A typical interaction flow proceeds in the following manner:
 
 ## Runtime implementation
 
+Here we demonstrate how self-contained Exe-Unit runtimes can be implemented, tested and finally plugged into the Provider's daemon.
+
+:warning: Please be aware that custom Exe-Unit runtime doesn't provide the level of isolation comparable with gvmi image. It's just executable running with the same privileges that daemon does.
+
+### Erigon runtime
+
+First, lets discuss what is the purpose of the Erigon runtime to gain some high level overview.
+
+* accepts the Ethereum network's name from the requestor agent,
+* starts Erigon binaries to sync with the chosen network and serve Erigon RPC endpoint,
+* generates credentials for basic authentication which is implemented with nginx
+* credentials are returned to the requestor agent so user of the service can connect to the Erigon RPC.
+
+Please refer to the "Provider's node" in the above Architecture overview diagram.
+
+### Implementation
+
+You might want to review simpler [example runtime](https://github.com/golemfactory/ya-runtime-sdk/blob/main/examples/example-runtime/src/main.rs) available in the [ya-runtime-sdk](https://github.com/golemfactory/ya-runtime-sdk/) repository.
+
+Erigon runtime declares two structures
+
+* `ErigonConf` - containing runtime configuration with default values,
+* `ErigonRuntime` - derives from `RuntimeDef` implements the required methods from `ya-runtime-sdk::Runtime` trait
+
+At least this methods from the `Runtime` trail has to be implemented but SDK provides default implementation to several others (TODO: link runtime SDK docs)
+
+* `deploy` - usually called before any command is send to the runtime, but it's possible to call it explicitly by agent SDKs.
```

Comment:
This sentence can be misunderstood. `deploy` and `start` are mandatory and always called in sequence